Logging docs 6.2 4.17 (#90136)
openshift/openshift-docs: modules/log6x-logging-loki-cli-install.adoc

  endpoint: https://s3.eu-central-1.amazonaws.com
  region: eu-central-1
----
<1> Use the name `logging-loki-s3` to match the name used in LokiStack.
<2> For the contents of the secret see the Loki object storage section.
+
--
include::snippets/logging-retention-period-snip.adoc[leveloffset=+1]
--

. Apply the `Secret` object by running the following command:
+
[source,terminal]
----
$ oc apply -f <filename>.yaml
----

. Create a `LokiStack` CR:
+
.Example `LokiStack` CR
[source,yaml]
----
apiVersion: loki.grafana.com/v1
kind: LokiStack
metadata:
  name: logging-loki # <1>
  namespace: openshift-logging # <2>
spec:
  size: 1x.small # <3>
  storage:
    schemas:
    - version: v13
      effectiveDate: "<yyyy>-<mm>-<dd>" # <4>
    secret:
      name: logging-loki-s3 # <5>
      type: s3 # <6>
  storageClassName: <storage_class_name> # <7>
  tenants:
    mode: openshift-logging # <8>
----
<1> Use the name `logging-loki`.
<2> You must specify `openshift-logging` as the namespace.
<3> Specify the deployment size. Supported size options for production instances of Loki are `1x.extra-small`, `1x.small`, or `1x.medium`. Additionally, `1x.pico` is supported starting with {logging} 6.1.

> <4> For new installations this date should be set to the equivalent of "yesterday", as this will be the date from when the schema takes effect.

Reviewer: Does "yesterday" need to be in quotes? 
```suggestion
<4> For new installations, use the date of the previous day, as this will be the date from when the schema takes effect.
```

<5> Specify the name of your log store secret.
<6> Specify the corresponding storage type.
<7> Specify the name of a storage class for temporary storage. For best performance, specify a storage class that allocates block storage. You can list the available storage classes for your cluster by using the `oc get storageclasses` command.
<8> The `openshift-logging` mode is the default tenancy mode where a tenant is created for log types, such as audit, infrastructure, and application. This enables access control for individual users and user groups to different log streams.

> 

Reviewer: Extra carriage return.
